environment.yml

```
@@ -16,7 +16,7 @@ dependencies:
   - pint
   - pip
   - pip:
-      - git+https://github.com/libra-project/libra-toolbox@v0.3
+      - git+https://github.com/libra-project/libra-toolbox@v0.7
```

Since libra-toolbox is now on pypi you can directly install it with:
```suggestion
      - libra-toolbox==0.7
```

---

Edit done!